eras/conway/impl/src/Cardano/Ledger/Conway/Transition.hs

```diff
@@ -12,8 +12,6 @@ module Cardano.Ledger.Conway.Transition (
   ConwayEraTransition (..),
   TransitionConfig (..),
   toConwayTransitionConfigPairs,
-  registerDelegs,
-  registerInitialDReps,
```

Comment:
Could you please add an entry to the changelog (for the latest version) that these functions have been removed